integrations/src/test/java/io/jenkins/plugins/casc/core/HudsonPrivateSecurityRealmConfiguratorTest.java

```diff
@@ -65,9 +68,11 @@ public void configure_all_attributes() {
         UserProperty mailerProperty = admin.getProperty(UserProperty.class);
         assertThat(mailerProperty.getEmailAddress(), is("admin3@example.com"));
 
-// Pending https://github.com/jenkinsci/ssh-cli-auth-module/pull/16
-//        UserPropertyImpl authorizedKeysProperty = admin.getProperty(UserPropertyImpl.class);
-//        assertThat(authorizedKeysProperty.authorizedKeys, is("ssh-rsa some-key\n"));
+        TimeZoneProperty timeZoneProperty = admin.getProperty(TimeZoneProperty.class);
+        assertThat(FieldUtils.readDeclaredField(timeZoneProperty, "timeZoneName", true), is("Europe/London"));
```

Comment:
I'm surprised it works without a getter for export that's quite weird, maybe just file a PR to Jenkins core?

Reply:
Indeed, I'm going to prepare one.

How to continue with the test in the meantime?

Reply:
use incremental from core build and we can integrate once baseline is above it

Reply:
Core updated to latest weekly version (includes the linked PR already).